src/views/Books/BookDetail.js

```diff
@@ -1,18 +1,30 @@
 import { useEffect, useState } from 'react';
+import { useParams, NavLink } from 'react-router-dom';
 import Book from '../../components/book/Book';
 import { getBookById } from '../../services/books';
 
+
 function BookDetail() {
-  const id = 1; // TODO: Use id from route
+  const params = useParams();
+  const id = params.id;
```

Comment:
Here we can destructure the id off of the params hook to make this a one liner: const { id } = useParams();